scripts/processDiagnosticMessages.ts

```diff
@@ -76,22 +76,13 @@ function buildInfoFileOutput(messageTable: InputDiagnosticMessageTable): string
 
 function buildDiagnosticMessageOutput(messageTable: InputDiagnosticMessageTable): string {
     let result = '{';
-    let first = true;
     messageTable.forEach(({ code }, name) => {
-        if (!first) {
-            first = false;
-        }
-        else {
-            result += ',';
-        }
-
         const propName = convertPropertyName(name);
-        result += `\r\n  "${createKey(propName, code)}": "${name.replace(/[\"]/g, '\\"')}"`;
+        result += `\r\n  "${createKey(propName, code)}" : "${name.replace(/[\"]/g, '\\"')}",`;
     });
 
-    result += '\r\n}';
-
-    return result;
+    // Shave trailing comma, then add newline and ending brace
+    return result.slice(0, result.length - 1) + '\r\n}';

```

Comment:
Only if we have zero diagnostics. Which we should never have. Or minimally we shouldn't still be generating this file if we do not.